lib/flutter_html.dart

```diff
@@ -49,6 +50,7 @@ class Html extends StatelessWidget {
   /// See [its wiki page](https://github.com/Sub6Resources/flutter_html/wiki/Style) for more info.
   Html({
     Key? key,
+    Key? keyAnchor,
```

Comment:
Rather than creating an extra parameter I would just suggest using the in-built `key` parameter for the `Html` widget. It makes a bit more sense and easier for users to implement. :)

I do like this idea though, if the user supplies a key themselves then they should be able to use that to create custom anchor implementations.

Reply:
@tneotia sorry not sure I understand. Right now the user can pass a key to HTML, however a GlobalKey is created for the HTMLParser.  

So essentially you can pass 2 keys to HTML, I know the naming is confusing but let me know if I am missing something.

Reply:
No problem, just suggesting to remove `keyAnchor` property and instead do `anchorKey = key ?? GlobalKey()`, since the `key` parameter can be used in the same way as the `keyAnchor`.

Reply:
Wouldn't that cause the same key to be used in the HTML and HTMLParser, and cause a "multiple widgets same globalkey" error?

Reply:
Ah yes I didn't realize that. What if you remove the `super(key: key)` line from the root `Html` widget and just keep it for `HtmlParser`?

Reply:
@tneotia I will make the change, and I understand the API is clunky like this but isn't there certain scenarios where they might want to attach a key to the parent element? (HTML).

Again, if you feel that is the way to go, is fine, just want to make sure I am not adding any breaking changes for existing users.

Reply:
To be honest I don't really know. I think @erickok 's expertise would be helpful here. I'm definitely not forcing you to make the change, just providing a suggestion/discussion if you will.

At it's core, `Html` only differs from `HtmlParser` by wrapping that widget in a `Container`. That's why I don't *think* there should be any breaking use case from this change, but like I said earlier I don't know for sure.

I would leave it as is for now and @erickok will perform his own review eventually so he will see this and comment if necessary. Thanks for the PR though! I like the idea.

Reply:
@leoafarias is right, it should be a separate key. It should probably also be forced to be a GlobalKey, as any other type (likely) won't work for the AnchorKey to work.

Reply:
So make it a GlobalKey. Personally I'd also rename to anchorKey rather than keyAnchor.